### PR TITLE
Add countdown timer to motions in actions list

### PR DESCRIPTION
**Pull request by chinins**
## Description

This PR adds countdown timer to a motion in the actions list

**New stuff** ✨

- add tiemoutPeriods to motions subscription, types & resolvers

**Changes** 🏗

- updated transformers accordingly
- update ActionsListItem css accordingly
- update CountdownTimer css accordingly
- cherry-picked Armando's commit for CountdownTimer copies
- updated FormattedAction type accordingly

Resolves DEV-398

Screenshot 2021-06-16 at 14 16 20

### CODE REVIEW

File: src/data/resolvers/motions.ts [hunk 1]
```diff
@@ -928,6 +928,44 @@ export const motionsResolvers = ({
         bigNumberify(fundamentalChainId),
       );
     },
+    async timeoutPeriods({
```
Comment (rdig) [inline src/data/resolvers/motions.ts:931]: Why did you add this logic in again ?

We already have it in the `motionTimeoutPeriods` resolver.

If you need to use it in a different resolver as well, just extract it somewhere you can use in both places, please don't duplicate code.

File: src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import React, {
   useCallback,
   useRef,
   MutableRefObject,
+  useMemo,
 } from 'react';
 import { FormattedMessage, defineMessage } from 'react-intl';
 import { useDispatch } from 'redux-react-hook';
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx:10]: There's a issue in this component, after the timeout runs out, it tries to trigger a motion values update (by calling `dispatch`)

But the `motionId` is just a `string`, at it's expecting a `bigNumber`

Screenshot from 2021-06-16 16-25-07

Please convert it to a `bigNumber` using `bigNumberify` before passing it to `dispatch`

**Review by rdig — CHANGES_REQUESTED** [review]
So in terms of functionality this works as expected.

Screenshot from 2021-06-16 16-28-31

The ellipsis display works as expected too:

Screenshot from 2021-06-16 16-30-54

However I am not happy with duplicating the resolver code, and I want you to fix this by extracting the logic and generalizing it so that we reduce code repetition.

Besides that, very nice work @chinins :100:

File: src/data/resolvers/motions.ts (hunk 1 above)
Comment (chinins) [reply src/data/resolvers/motions.ts:931]: I did it for testing and I was meaning to extract it but then forgot

Comment (chinins) [reply src/data/resolvers/motions.ts:931]: I've done it now and you can rereview.

**Review by rdig — COMMENTED** [review]
Good to go! Very nice work :100: 

PS: Make sure to rebase your branch on `feature/reputation-voting` since your colonyNetwork version is behind

Screenshot from 2021-06-17 20-15-47

**Review by rdig — APPROVED** [review]
Damn it Github! I wanted to approve it!
